firebase.json

```diff
@@ -48,6 +48,56 @@
           "destination": "https://zksync.io/contact",
           "type": "302"
         },
+        {
```

Comment:
this is duplicate with the main branch (there are already two for prod and staging)